app/presenters/offender_with_allocation_presenter.rb

```diff
@@ -29,4 +29,8 @@ def allocation_date
       (@allocation.primary_pom_allocated_at || @allocation.updated_at)&.to_date
     end
   end
+
+  def primary_pom_allocated_at
```

Comment:
Reintroducing this method back, as it is needed to sort by this column in certain pages. Was deleted by mistake.